backend/src/main/java/com/fydp/backend/model/Summary.java

```diff
@@ -0,0 +1,89 @@
+package com.fydp.backend.model;
+
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+//model to interact with Summary table
+@Entity
+
+@Table(name = "summary", schema = "brevity")
+public class Summary {
+
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long summary_id;
```

Comment:
Does this have to be `Long` over `long`? same for `chpId`

Reply:
not sure if it needs to be , but it seems like it is the convention in javax.persistence for PKs

https://docs.oracle.com/javaee/7/api/javax/persistence/GeneratedValue.html